mava/systems/jax/config.py

```diff
@@ -223,8 +226,9 @@ def get_local_config(self, component: Type[Component]) -> SimpleNamespace:
         global_config = self._built_config
         local_config: Dict[str, Any] = {}
 
-        config_class = component.config_class()
-        if not config_class:  # no config class for component
+        config_class = component.__init__.__annotations__["config"]
+
+        if config_class is SimpleNamespace:  # no config class for component
```

Comment:
This if statement is redundant though I get that you want to return before the for loop

Reply:
The code is to return before the loop if necessary, yes. Is there a better way of writing this that you can think of, @AsadJeewa? I updated the comments and code layout.

Reply:
The latest change is better